wfl/select/flat_histogram.py

```diff
@@ -101,6 +101,7 @@ def _select_indices_flat_boltzmann_biased(quantities, n, kT=-1.0, bins='auto', b
         number of samples to return
     kT: float, default -1
         if > 0 temperature to bias by
+        [kT] should have the same unit as the property quantities
```

Comment:
Almost.  I'm happy with the 1st change, because "quantities" is a _parameter_ to the (internal) function.  However, I prefer the old wording for the 2nd place you changed, because "info_field" is a string indicating which `Atoms.info` property to use.